lib/radar.js

```diff
@@ -12,14 +12,18 @@ const headers = {
 }
 
 const defaults = {
-	FAA: true, // use US/Canada radar data source
-	FLARM: true, // use FLARM data source
-	MLAT: true, // use MLAT data source
-	ADSB: true, // use ADS-B data source
-	inAir: true, // fetch airborne aircraft
-	onGround: false, // fetch (active) aircraft on ground
-	inactive: false, // fetch inactive aircraft (on ground)
-	gliders: false // fetch gliders
+	// see: https://www.flightradar24.com/how-it-works
+	FAA:   		true, 	// use US/Canada radar data source
```

Comment:
Make the key [camel-cased](https://en.wikipedia.org/wiki/Camel_case). Same for `FLARM`, `MLAT`, `ADSB`, `ESTIMATED`.